app/views/mentors/_mentor_form.html.erb

```diff
@@ -1,7 +1,9 @@
 <%= simple_form_for(locals[:mentor], wrapper: :horizontal_form, html: {class: 'form-horizontal'},
                     url: {action: (locals[:is_new] ? 'create' : 'update')}) do |f| %>
   <%= f.error_notification %>
-  <%= f.input :user_id, collection: locals[:users].map {|user| [user.user_name, user.id]}, include_blank: false, required: true %>
+  <% if locals[:is_new] %>
+    <%= f.input :user_id, collection: locals[:users].map {|user| [user.user_name, user.id]}, include_blank: false, required: true %>
+  <% end %>
```

Comment:
This is hidden if we're just editing; in most use cases, we should not be changing which user the user role is assigned to.